.github/workflows/pr.yaml

```
@@ -53,28 +54,30 @@ jobs:
 
       - name: Terraform Plan
         env:
-          TF_VAR_portainer_url: ${{ secrets.PORTAINER_URL }}
-          TF_VAR_portainer_api_key: ${{ secrets.PORTAINER_API_KEY }}
           TF_VAR_github_registry_username: ${{ github.actor }}
           TF_VAR_github_registry_access_token: ${{ secrets.GITHUB_TOKEN }}
-          TF_VAR_terramagma_local_ip: ${{ secrets.TERRAMAGMA_LOCAL_IP }}
-          PG_CONN_STR: ${{ secrets.POSTGRES_CONNECTION_STRING }}
+          TF_VAR_gcp_project_id: ${{ secrets.GCP_PROJECT_ID }}
+          TF_VAR_google_service_account_id: ${{ secrets.GOOGLE_SERVICE_ACCOUNT_ID }}
+          TF_VAR_app_image: ghcr.io/terra-magma/terrastack-docs:${{ github.sha }}.${{ steps.timestamp.outputs.container-tag }}
```

actionlint (1.7.10)

[error] property "timestamp" is not defined in object type {}

(expression)